README.rst

```diff
@@ -286,6 +287,9 @@ keys:
 
     .. _DataLad: https://www.datalad.org
 
+.. _Python regular expressions: https://docs.python.org/3/library/re.html
+                                #regular-expression-syntax
```

Comment:
note to myself: tested in github README.rst renderer for the diff -- it manages to bring the `#fragment` properly into the url